docs/release-notes/index.rst

```diff
@@ -0,0 +1,257 @@
+.. _release-notes:
+
+Release notes
+=============
+
+This page lists past release notes for Rockcraft, summarising new features, bug
+fixes and backwards-incompatible changes in each version. It also contains the
+release and support policies for Rockcraft.
+
+
+Current releases
+----------------
+
+
+Rockcraft 1
```

Comment:
@medubelko is this the desired format of the "<latest release>" header?

Reply:
If that's the title for this mainline release, yes. I think for Snapcraft we call it "Snapcraft  7".

Reply:
There hasn't been a title yet because there's only ever been this one "mainline" release, but "Rockcraft 1" is as good as any

Reply:
...although the other option, in this case, would be to omit this header (since there's only one) and re-introduce it once "Rockcraft 2" comes around?

Reply:
@medubelko I ended up doing this^. Let me know if you want me to add the "Rockcraft 1" header back.

Reply:
I think that works.